src/Symfony/Component/Routing/Loader/Configurator/RoutingConfigurator.php

```diff
@@ -36,7 +36,7 @@ public function __construct(RouteCollection $collection, PhpFileLoader $loader,
     /**
      * @return ImportConfigurator
      */
-    final public function import($resource, $type = null, $ignoreErrors = false)
+    final public function import($resource, $type = null, $ignoreErrors = PhpFileLoader::IGNORE_NONE)
```

Comment:
the constant is in FileLoader.

Reply:
and this requires bumping the min requirement.

Reply:
Min. requirements changed in src/Symfony/Component/Routing/composer.json: `"symfony/config": "^4.4|^5.0",`